imops/interp2d.py

```diff
@@ -90,16 +90,16 @@ def __call__(self, points: np.ndarray, values: np.ndarray = None, fill_value: fl
         new_values: np.ndarray
             interpolated values at given points
         """
-        self.values = values or self.values
+        x_values = values if values is not None else self.values
```

Comment:
why not just 
```python
if values is None:
  values = self.values
```
? the new variable seems redundant

Reply:
fixed